nms/packages/magmalte/app/views/subscriber/SubscriberAddDialog.js

```diff
@@ -13,6 +13,8 @@
  * @flow strict-local
```

Comment:
Split this up into different files, there are quite a few components here, and the file itself is incredibly long.